### PR TITLE
Small cleanings and polishings

### PR DESCRIPTION
**Pull request by flosincapite**

### CODE REVIEW

**Comment by xmunoz** [comment]
Since this PR is growing much larger than expected, it would be helpful if you could summarize the changes in the PR description, so I know what to expect and test for in the next round of reviews.

File: engine/actor.py [hunk 1]
```diff
@@ -250,18 +238,19 @@ def upon_death(self, callback):
...
+    newborn = Actor(
         actor_item,
         Health(health),
         Psyche(psyche),
         Strength(strength),
         Stamina(stamina),
         Will(will),
         Wisdom(wisdom),
-        Insanity(insanity),
+        Insanity(100),
```
Comment (xmunoz) [inline engine/actor.py:257]: Does this mean that the initial value of insanity is 100? If so, why?

File: engine/say.py [hunk 2]
```diff
@@ -64,7 +64,7 @@ def insayne(text, add_newline=True, insanity=None):
     if add_newline:
         adventurelib.say("")
     if insanity is None:
-        insanity = G.player.insanity.value
+        insanity = G.player.insanity.current_value
```
Comment (xmunoz) [inline engine/say.py:67]: The distinction between `value` and `current_value` isn't clear to me. Could you choose different variable names that elucidate the difference between these values?

**Review by xmunoz — CHANGES_REQUESTED** [review]
I also don't understand why `modify` was re-named to `heal_or_harm`. My general review is that it's hard to understand what's going on in this PR, likely because of poor naming choices.

**Comment by flosincapite** [comment]
I will break this up into some smaller PRs.

**Comment by flosincapite** [comment]
This change is now subsumed by changes #88 , #90 , #91 , and #92 , q.v.

**Pull request closed without merging**